doc/users/installing/install.rst

```diff
@@ -0,0 +1,3 @@
+.. redirect-from:: /users/installing
```

Comment:
This leaves /users/installing/index.html a blank page except for the heading "Installation"  Did we want to un-hide the toctree?

Reply:
https://65000-1385122-gh.circle-artifacts.com/0/doc/build/html/users/installing/index.html

Reply:
Hmm yea, that's annoying. The problem is that including `INSTALL.rst` directly causes duplicate labels and other weirdness. I guess showing the toc might be the easiest fix.

Reply:
Yes, I think a problem here is the include.  Seems like it would be preferable for INSTALL.rst to simply refer to the correct place in the documentation, rather than have canonical information.

Reply:
Alright, I ended up moving things into `doc/users/installing.rst` for multiple reasons. Having it separate caused weirdness in the toctree of the PDF, moving it to `INSTALL.rst` breaks the redirects, and putting things in `doc/users/installing/install.rst` like I did before makes empty pages without tocs.

Reply:
Is there a reason to not have it in `doc/users/installing/index.rst`?  Other than the fact that maybe installing doesn't have any subfiles now?

Reply:
It also used to be there in <=3.4, so it seemed best to move it back with it being only one file.

Reply:
Yes, but moving it was deliberate.  Currently from https://matplotlib.org/stable/users/index under "General" each of the sections (Getting Started, Installation, etc) goes to its own subdirectory in `users/getting_started` etc.

Reply:
OK, I can revert that. I wonder if we could end up keeping the file split, if we _only_ got rid of the `include` stuff, but this is probably fine.